index.html

```diff
@@ -233,7 +256,7 @@
         e.preventDefault();
         return false;
     }
-    else if (e.keyCode == 68 && e.ctrlKey) {    // CTRL + D
+    else if (e.keyCode == 68 && e.ctrlKey && !e.shiftKey) {    // CTRL + D

```

Comment:
In the latest version (merged now) there is also support for Mac CMD key with meta key. Can you update here?